src/SBVonKarman.cpp

```diff
@@ -160,7 +162,10 @@ namespace galsim {
         dbg<<"_delta = "<<_delta<<'\n';
 
         // build the radial function, and along the way, set _stepk, _hlr.
-        _buildRadialFunc();
+        if (force_stepk == 0.0)
+            _buildRadialFunc();
```

Comment:
Is there really any advantage to this?  
Why not just `_stepk = force_stepk` always and let it be built whenever it is needed?